deployments/puppet/README.md

```diff
@@ -0,0 +1,142 @@
+  on the node.  The `SPLUNK_BUNDLE_DIR` environment variable will be set to
+  this value for the collector service.  (**default:**
+  `/usr/lib/splunk-otel-collector/agent-bundle`)
+
+- `$splunk_collectd_dir`: The path to the collectd config directory for the
+  Smart Agent bundle.  The default path is provided by the collector package.
+  If the specified path is changed from the default value, the path should be
+  an existing directory on the node.  The `SPLUNK_COLLECTD_DIR` environment
+  variable will be set to this value for the collector service.  (**default:**
+  `${splunk_bundle_dir}/run/collectd`)
+
+- `$service_user` and `$service_group` (Linux only): Set the user/group
+  ownership for the collector service. The user/group will be created if they
+  do not exist.  (**default:** `splunk-otel-collector`)
+
+- `$with_fluentd`: Whether to install/manage fluentd and dependencies for log
+  collection.  The dependencies include [capng_c](
+  https://github.com/fluent-plugins-nursery/capng_c) for enabling
+  [Linux capabilities](
+  https://docs.fluentd.org/deployment/linux-capability),
+  [fluent-plugin-systemd](
+  https://github.com/fluent-plugin-systemd/fluent-plugin-systemd) for systemd
+  journal log collection, and the required libraries/development tools.
+  (**default:** `true`)
+
+- `$fluentd_config_source`: Source path to the fluentd config file.  This file
+  will be copied to the `$fluentd_config_dest` path on the node.
+  See the [source attribute](
+  https://puppet.com/docs/puppet/latest/types/file.html#file-attribute-source)
+  of the `file` resource for supported value types.  The default source file is
+  provided by the collector package. (**default:**
+  `file:///etc/otel/collector/fluentd/fluent.conf`)
+
+- `$fluentd_config_dest`: Destination path to the fluentd config file on the
+  node.  (**default:** `/etc/otel/collector/fluentd/fluent.conf`)
+
+- `$manage_repo` (Linux only): In cases where the collector and fluentd apt/yum
+  repositories are managed externally, set this to `false` to disable
+  management of the repositories by this module.  **Note:** If set
+  to `false`, the apt (`/etc/apt/sources.list.d/splunk-otel-collector.list` and
+  `/etc/apt/sources.list.d/splunk-td-agent.list`) and yum
+  (`/etc/yum.repos.d/splunk-otel-collector.repo` and
+  `/etc/yum.repos.d/splunk-td-agent.repo`) repository definition files will be
+  deleted if they exist in order to avoid any conflicts.  (**default:** `true`)
+
+## Dependencies
+
+On Linux-based systems, the
+[puppetlabs/stdlib](https://forge.puppet.com/puppetlabs/stdlib) module is
+required.
+
+On Debian-based systems, the
+[puppetlabs/apt](https://forge.puppet.com/puppetlabs/apt) module is required to
+manage the collector and fluentd apt repositories.
+
+On RPM-based systems, the
+[puppet/yum](https://forge.puppet.com/puppet/yum) module is required to
+install the "Development Tools" package group as a dependency for fluentd.
+
+## Release Process
```

Comment:
Are all of these steps required as part of the release process? If so, I would suggest maybe making this a list of steps so that the user knows which things to do and in which order.